main.js

```diff
@@ -77,20 +77,20 @@ const log = report => {
 	process.exit(report.errorCount === 0 ? 0 : 1);
 };
 
+const files = (report, predicate) => report.results
+	.filter(predicate)
```

Comment:
> I had the assumption that the warnings should only open if there er no errors.

Aww sorry, yes that make sense. Nevertheless, I think sorting _additionally_, would be reasonable. Would you mind adding it? No problem, if you don't, I could take care of it too.